lecture/src/main/docker/app.yml

```diff
@@ -0,0 +1,43 @@
+# This configuration is intended for development purpose, it's **your** responsibility to harden it for production
+version: '3.8'
+services:
+    lecture-app:
+        image: lecture
+        environment:
+            - _JAVA_OPTIONS=-Xmx512m -Xms256m
+            - SPRING_PROFILES_ACTIVE=prod,api-docs,artemis
+            - MANAGEMENT_METRICS_EXPORT_PROMETHEUS_ENABLED=true
+            - EUREKA_CLIENT_SERVICE_URL_DEFAULTZONE=http://admin:$${jhipster.registry.password}@jhipster-registry:8761/eureka
+            - SPRING_CLOUD_CONFIG_URI=http://admin:$${jhipster.registry.password}@jhipster-registry:8761/config
+            - SPRING_DATASOURCE_URL=jdbc:mysql://lecture-mysql:3306/lecture?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC&createDatabaseIfNotExist=true
+            - SPRING_LIQUIBASE_URL=jdbc:mysql://lecture-mysql:3306/lecture?useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC&createDatabaseIfNotExist=true
+            - JHIPSTER_SLEEP=30 # gives time for other services to boot before the application
+        depends_on:
+            - artemis-mysql
+    artemis-mysql:
+        image: mysql:8.0.25
+        # volumes:
+        #   - ~/volumes/jhipster/lecture/mysql/:/var/lib/mysql/
+        environment:
+            - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+            - MYSQL_DATABASE=lecture
+            - MYSQL_DATABASE=Artemis
+        # If you want to expose these ports outside your dev PC,
+        # remove the "127.0.0.1:" prefix
+        ports:
+            - 127.0.0.1:3306:3306
```

Comment:
Double port? Same below

Reply:
yes, the one is the target port inside of the container, the other one is the one exposed for the outer world